src/main/java/com/icthh/xm/ms/entity/domain/UniqueField.java

```diff
@@ -31,13 +35,17 @@
     @SequenceGenerator(name = "sequenceGenerator")
     private Long id;
 
+    @Column(name = "field_json_path")
     private String fieldJsonPath;
 
+    @Column(name = "field_value")
```

Comment:
should be `nullable=false` from `20180511124100_added_entity_unique_fields.xml`

Reply:
fiexed